### PR TITLE
Updated adiabatic conditions to include chunk geometry

### PR DESCRIPTION
**Pull request by bobmyhill**

### CODE REVIEW

**Comment by tjhei** [comment]
Can you please add a test that executes this section of code? /run-tests

**Comment by bobmyhill** [comment]
I'm happy with this now, but can change more, or add another test in 3d if you want :)

**Comment by bobmyhill** [comment]
Ok, 3D test added.

File: include/aspect/geometry_model/chunk.h [hunk 1]
```diff
@@ -117,6 +117,17 @@ namespace aspect
         virtual
         double longitude_range() const;
 
+
+        virtual
+        double start_latitude() const;
```
Comment (bangerth) [inline include/aspect/geometry_model/chunk.h]: can you document this and the other similar functions? Would it be worthwhile calling them `south_latitude`, `north_latitude` etc to make it more self-explanatory?

Comment (bangerth) [inline include/aspect/geometry_model/chunk.h]: please also state the units (radians).

**Comment by bangerth** [comment]
Just a very small number of comments. Can you update, squash, and re-upload?

**Comment by bobmyhill** [comment]
Ok, updated and squashed :)

**Comment by bangerth** [comment]
Thanks. Looks good to me as well.

@tjhei : Same procedure?

**Comment by bangerth** [comment]
@tjhei ?

**Comment by tjhei** [comment]
It looks like the patch generation is now working. Bob, do you know how to apply http://www.math.clemson.edu/~heister/aspect-logs/9eb9a57048fd31a88b3ccefd48dd55ec5f6b8797/changes.diff ?

**Comment by bobmyhill** [comment]
I think I've made the changes - let's see what the tester thinks!

**Comment by bobmyhill** [comment]
Oh, I think she (?he) heard me! The tester thinks happy thoughts... :)

File: include/aspect/geometry_model/chunk.h [hunk 2]
```diff
@@ -108,21 +108,66 @@ namespace aspect
         virtual
         Point<dim> representative_point(const double depth) const;
 
+        /**
+               * Return the longitude at the western edge of the chunk
```
Comment (tjhei) [inline include/aspect/geometry_model/chunk.h]: This looks really weird. Is this a mixture of tabs and spaces or something? Why does astyle not fix this?

Comment (bangerth) [inline include/aspect/geometry_model/chunk.h]: Does astyle really fix up comment indentation? In any case, @bobmyhill , can you just adjust this by hand?

**Comment by bobmyhill** [comment]
docs/indent breaks the indentation during the conversion from tabs to spaces (for me, at least). 
Fixed manually, and I'll check more carefully in future. Thanks!

**Pull request merged**